From: Rinigus <rinigus.git@gmail.com>
Date: Tue, 28 May 2019 20:35:53 +0300
Subject: use qt-based http file source

---
 mapbox-gl-native-lib.pro | 17 +++++++----------
 1 file changed, 7 insertions(+), 10 deletions(-)

diff --git a/mapbox-gl-native-lib.pro b/mapbox-gl-native-lib.pro
index a4941f8..90ec3b5 100644
--- a/mapbox-gl-native-lib.pro
+++ b/mapbox-gl-native-lib.pro
@@ -4,11 +4,8 @@ TEMPLATE = lib
 CONFIG += qt c++14 exceptions warn_off object_parallel_to_source
 CONFIG += use_system-zlib use_icu
 
-# QT += network \
-#       gui \
-#       sql
-
-QT += gui \
+QT += network \
+      gui \
       sql
 
 # QMAKE_CXX=/opt/gcc6/bin/g++
@@ -72,8 +69,8 @@ use_icu {
 
 SOURCES += \
     platform/qt/src/async_task.cpp \
-\ #    platform/qt/src/http_file_source.cpp \
-\ #    platform/qt/src/http_request.cpp \
+    platform/qt/src/http_file_source.cpp \
+    platform/qt/src/http_request.cpp \
     platform/qt/src/qmapbox.cpp \
     platform/qt/src/qmapboxgl.cpp \
     platform/qt/src/qmapboxgl_map_observer.cpp \
@@ -391,14 +388,14 @@ SOURCES += \
     vendor/nunicode/src/libnu/tounaccent.c \
     vendor/nunicode/src/libnu/toupper.c \
     vendor/nunicode/src/libnu/utf8.c \
-    platform/default/http_file_source.cpp
+  # platform/default/http_file_source.cpp
 
 HEADERS += \
     platform/qt/include/qmapbox.hpp \
     platform/qt/include/qmapboxgl.hpp \
     platform/qt/src/async_task_impl.hpp \
-\#    platform/qt/src/http_file_source.hpp \
-\#    platform/qt/src/http_request.hpp \
+    platform/qt/src/http_file_source.hpp \
+    platform/qt/src/http_request.hpp \
     platform/qt/src/qmapboxgl_map_observer.hpp \
     platform/qt/src/qmapboxgl_map_renderer.hpp \
     platform/qt/src/qmapboxgl_p.hpp \
